tests/pages/dashboardPage.js

```diff
@@ -1015,6 +1017,11 @@ module.exports = {
       'ReplSet Last Election',
       'MongoDB Versions',
     ],
+    elements: {
+      replicationLagMin: (serviceName) => locate(`//div[@aria-label="Replication Lag panel"]//a[@title="${serviceName}"]//ancestor::tr[@class="graph-legend-series "]//td[@class="graph-legend-value min"]`),
```

Comment:
hi @peterSirotnak , this comes a bit late, but since PR is not merged perhaps you want to look at some suggestions provided by @vyarosh on https://github.com/percona/pmm-ui-tests/pull/728/files and use graphLegendSeriesValue here?